cloudformation/lecture13_Security.yml

```
@@ -0,0 +1,122 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: RaiseTech-lecture13-Security template
+
+Parameters:
+  NameBase:
+    Description: this is base name.
+    Type: String
+    Default: RaiseTech-lecture13
+
+Resources: 
+# EC2のSecurityGroupの作成
+  EC2SG:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: lecture13-EC2-SG
+      VpcId: !ImportValue RaiseTech-lecture13-VPC-ID
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 0.0.0.0/0 # !Ref MyIP 課題用に変更
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 3000
+          ToPort: 3000
+          CidrIp: 0.0.0.0/0
+      Tags:
+        - Key: Name
+          Value: !Sub ${NameBase}-EC2SG
+
+# RDSのSecurityGroupの作成
+  RDSSG:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: lecture13-RDS-SG
+      VpcId: !ImportValue RaiseTech-lecture13-VPC-ID
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 3306
+          ToPort: 3306
+          SourceSecurityGroupId: !Ref EC2SG
+      Tags:
+        - Key: Name
+          Value: !Sub ${NameBase}-RDSSG
+
+# ALBのSecurityGroupの作成
+  ALBSG:
+      Type: AWS::EC2::SecurityGroup
+      Properties:
+        GroupDescription: lecture13-ALB-SG
+        VpcId: !ImportValue RaiseTech-lecture13-VPC-ID
+        SecurityGroupIngress:
+          - IpProtocol: tcp
+            FromPort: 80
+            ToPort: 80
+            CidrIp: 0.0.0.0/0
+          - IpProtocol: tcp
+            FromPort: 3000
+            ToPort: 3000
+            CidrIp: 0.0.0.0/0
+        Tags:
+          - Key: Name
+            Value: !Sub ${NameBase}-ALBSG
+
+# IAMロールの作成
+  IAMRole:
```

論理IDが統一されていないのが気になりました！他と統一するなら、
```suggestion
  IAMROLE:
```
になるかなと！